src/spaces/p3-linear.js

```diff
@@ -14,6 +14,7 @@ const fromXYZ_M = [
 
 export default new RGBColorSpace({
 	id: "p3-linear",
+	cssId: "--display-p3-linear",
```

Comment:
Oof. I understand why it's as awkward as it is, but it makes me sad.

I wonder if it may make sense to allow providing multiple dashed ids, so we could also provide `--p3-linear` but that's a out of scope for this PR…

Reply:
we should not have called it just "p3" because that is ambiguous. Cinema uses DCI-P3, video uses PR-D65 and CSS (among other things) uses Display-P3.